### PR TITLE
Update README.md test

### PR DESCRIPTION
**Pull request by k0shk0sh**
Hello world

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

### CODE REVIEW

**Review by FastAccess — COMMENTED** [review]
## Code

```
if (!InputHelper.isEmpty(review.getBody())) {
    body.setVisibility(View.VISIBLE);
    HtmlHelper.getComment(body, review.getBody());
} else {
    body.setVisibility(View.GONE);
}
```

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # testing_pullrequest
```
Comment (k0shk0sh) [inline README.md:1]: hello

**Comment by FastAccess** [comment]
@k0shk0sh Hello

**Comment by FastAccess** [comment]
Hello?

**Comment by FastAccess** [comment]
hello

**Pull request closed without merging**